src/components/SnackStack.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, Snackbar } from '@mui/material';
+import { useSnackStack } from './SnackStackProvider';
+import SnackbarToast from './SnackToast';
+
+function SnackStack() {
+  const { toastsPack } = useSnackStack();
+
+  const firstToast = toastsPack[0];
+
+  return (
+    <Snackbar
+      open={!!firstToast}
```

Comment:
is this a typo? Looks like double negative?

Reply:
It's one way to cast to boolean https://stackoverflow.com/questions/10467475/double-negation-in-javascript-what-is-the-purpose